packages/taquito-michelson-encoder/test/tokens/timestamp.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { TimestampToken } from '../../src/tokens/comparable/timestamp';
+
+describe('Timestamp token', () => {
+  let token: TimestampToken;
+
+  beforeEach(() => {
+    token = new TimestampToken({ prim: 'timestamp', args: [{ string: '1638554817'}], annots: [] }, 0, null as any);
```

Comment:
The args should be empty here as the token only represents the Michelson type, not the value.

Reply:
oops nice catch, I meant them to be empty.